code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -105,7 +105,7 @@
 
 		update_muts=1
 
-	if(!istype(src, /mob/living/carbon/monkey/diona))
+	if(type == /mob/living/carbon/monkey)
```

Comment:
Why did you change this?

This check is failing as-is by the way, because the type variable is text.